src/neo/SmartContract/Native/NonfungibleToken.cs

```diff
@@ -137,7 +137,7 @@ protected IIterator TokensOf(DataCache snapshot, UInt160 owner)
             return new ArrayWrapper(tokens.Select(p => (StackItem)p).ToArray());
         }
 
-        [ContractMethod(CpuFee = 1 << 17, StorageFee = 50, RequiredCallFlags = CallFlags.WriteStates | CallFlags.AllowNotify)]
+        [ContractMethod(CpuFee = 1 << 17, StorageFee = 50, RequiredCallFlags = CallFlags.States | CallFlags.AllowNotify)]
```

Comment:
Not directly related, but I think `AllowCall` is missing here for proper `onNEP11Payment`.

Reply:
Yes, we should add it

Reply:
Fixed.